utils/python/CIME/SystemTests/nck.py

```diff
@@ -107,6 +107,7 @@ def run(self):
         #======================================================================
 
         if success:
+            os.remove("%s/%s.exe" % (exeroot, cime_model))
```

Comment:
The copy below was failing on certain systems due to "file busy" type errors. I found that removing the file before the copy prevented these errors for reasons that are not entirely clear to me.